students/Jaidjen/lesson04/Mailroom2.py

```diff
@@ -1,77 +1,103 @@
 import sys  # imports go at the top of the file
 
 
-Mail = {1:{'Name':'James Smith', 'Donation':[2000.25,500,27.50]},
-              2:{'Name': 'Dorothy Parker','Donation':[3175,475.45,92.30]},
-              3:{'Name':'Dohgyu Hwang','Donation':[10670,350.19,175]},
-              4:{'Name': 'Sally Yu', 'Donation':[100,925]},
-              5:{'Name':'Jenny Ramos','Donation':[25.14]}}
-
-prompt = "\n".join(("Functions:",
-          "Please choose from below options:",
-          "1 - Send a Thank You to a single donor",
-          "2 - Create a Report",
-          "3 - Send letters to all donors",
-          "4 - Quit",
-          ">>> "))
-
-
-def view_mail(Mail):
+donors = [{'Name':'James Smith', 'Donation':[21000.25,500,27.50]},
+          {'Name': 'Dorothy Parker','Donation':[3175,475.45,92.30]},
+          {'Name':'Dohgyu Hwang','Donation':[10670,350.19,175]},
+          {'Name': 'Sally Yu', 'Donation':[100,925]},
+          {'Name':'Jenny Ramos','Donation':[25.14]}]
```

Comment:
you were using a dictionary before, now you're using a list.

this is what your data structure should look like:
```
donors = {"James Smith": [100, 200, 300], "Sally Yu": [202, 22.3]}
```


once you convert this to above dictionary then you can update your look up and update code (it will get much simpler)